lib/stack/events.ts

```diff
@@ -0,0 +1,12 @@
+export const eventTipChannel = (id: string) => `channel_tip_${id}` as const;
+export const eventTipCashback = (fid: number) => `tip_cashback_${fid}` as const;
+export const eventTipRecipient = (fid: number) => `tip_recipient_${fid}` as const;
+
+export const eventStakeChannel = (id: string) => `channel_stake_${id}`;
+export const eventStakeChannelFid = (id: string, fid: number) => `channel_stake_${id}_${fid}` as const;
```

Comment:
this is an `unstake` event right?
- NOTES sent from `channel` to `fid` for an `unstake` event
is that correct?

Reply:
if yes, can we refactor this event name to `eventUnstakeChannel` so it's easier to understand?

Reply:
This event is used only for user stakes/unstakes. It helps track staking user stakes by channelId.

Reply:
so this event is for `BOTH` staking & unstaking? (bidirectional event name)
is that correct?